embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/SchemaType.kt

```diff
@@ -10,7 +10,7 @@ import io.embrace.android.embracesdk.payload.AppExitInfoData
 
 internal object SchemaKeys {
     internal const val CUSTOM_BREADCRUMB = "emb-custom-breadcrumb"
-    internal const val VIEW_BREADCRUMB = "view-breadcrumb"
+    internal const val VIEW_BREADCRUMB = "screen-view"
```

Comment:
Would be good to be consistent with the usage of the `emb-` prefix for these keys.

Reply:
If these are the canonical name of the telemetry generated by this schema, we should look to define them in such a away that we don't have to attach the prefix with each. With that, we can have one place where we decide whether to do it.

Reply:
Agreed. Fixing this here one way or the other for now would be good, but long term, we should really define the canonical names sans the prefix and just attach them then necessary.

Reply:
I'll skip this for now as this PR has been pretty held up already, but will address in a future changeset (if Hanson's schema type changes don't already address this)